gt-scheduler/website
# Update sort algorithm for "most compact"

### Summary

Resolves #160 

Updates the "most compact" sorting option to account for custom blocks

### Checklist

- [x] Combinations are sorted appropriately for each setting.
- [x] Related components (if any) are updated if this change affects them (`Calendar` is an example). -- I don't think anything was impacted by this change

### How to Test
Create a few custom blocks, check how the ordering of the combinations changes as a result
